workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -43,6 +150,11 @@ public static int RunIn(string workingDirectory, string command, params string[]
                 WorkingDirectory = workingDirectory
             };
 
+            foreach (var envVarEntry in envVars ?? new Dictionary<string, string>())
```

Comment:
Why not just wrap the `foreach` in a null check rather than allocate a new Dictionary?